alpaca/trading/client.py

```diff
@@ -591,7 +591,7 @@ def get_corporate_annoucements(
         """
         params = filter.to_request_fields() if filter else {}
 
-        if "ca_types" in params and type(params["ca_types"]) is List:
+        if "ca_types" in params and params["ca_types"]:
```

Comment:
Just a curious (I'm not a Python expert) why did you remove the type check at all in this case? It looks like an error to me - most probably we need the same `isinstance` check here.

Reply:
You're right that this is an error, I should have checked type as you've pointed out. I'll fix this now, thank you 🤝